### PR TITLE
[SPARK-30247[[PySpark][FOLLOWUP] Add Python class MultivariateGaussian

### PR DESCRIPTION
**Pull request by huaxingao**
### What changes were proposed in this pull request?
add a corresponding class MultivariateGaussian containing a vector and a matrix on the py side, so gaussian can be used on the py side.

### Does this PR introduce any user-facing change?
add Python class ```MultivariateGaussian```

### How was this patch tested?
doctest

### CODE REVIEW

**Comment by huaxingao** [comment]
@srowen @zhengruifeng

**Review by srowen — APPROVED** [review]
Looks good if it's for consistency

**Review by zhengruifeng — APPROVED** [review]

**Comment by zhengruifeng** [comment]
Merged to master, thanks all!

**Comment by huaxingao** [comment]
Thanks! @zhengruifeng @srowen

File: python/pyspark/ml/stat.py [hunk 1]
```diff
@@ -394,6 +395,22 @@ def summary(self, featuresCol, weightCol=None):
         return Column(self._java_obj.summary(featuresCol._jc, weightCol._jc))
 
 
+class MultivariateGaussian(object):
```
Comment (zero323) [inline python/pyspark/ml/stat.py]: Just a thought - `namedtuple` would be more than sufficient here.

**Pull request closed without merging**